webview/src/experiments/components/table/styles.module.scss

```diff
@@ -657,6 +669,10 @@ $badge-size: 0.85rem;
   @extend %iconBox;
 
   padding: 1px;
+
+  &[aria-disabled='true'] {
+    cursor: default;
```

Comment:
Why not adjust the opacity here so we don't have to add two `[aria-disabled='true']` blocks for the button in our styles?